README.md

```diff
@@ -5,10 +5,7 @@
 Code: [github](https://github.com/geostyler/geostyler)
 Package: [npm](https://www.npmjs.com/package/geostyler)
 
-Documentation:
-[main](https://geostyler.github.io/geostyler/main/index.html) /
-[latest](https://geostyler.github.io/geostyler/latest/index.html) /
-[11.0.0](https://geostyler.github.io/geostyler/v11.0.0/index.html)
+Documentation: [main](https://geostyler.github.io/geostyler/main/index.html) / [latest](https://geostyler.github.io/geostyler/latest/index.html) / [11.0.0](https://geostyler.github.io/geostyler/v11.0.0/index.html)
```

Comment:
Could you add a comment above that following line will be replaced automaticallly?